frontend/workflows/projectCatalog/src/details/header.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Grid, styled, Typography } from "@clutch-sh/core";
+import { capitalize } from "lodash";
+
+interface ProjectHeaderProps {
+  name: string;
+  route?: string;
+  routeTitle?: string;
+  description?: string;
+}
+
+const TextLink = styled("a")({
+  textDecoration: "none",
+  color: "unset",
+});
+
+const ProjectHeader = ({
+  name,
+  route = "/catalog",
```

Comment:
It would be nice if we could read this in from the config. I think maybe in the immediate we might not want this to be clickable.

Reply:
Yeah that was the original idea, I'll remove the linking for MVP